Lib/test/support/hashlib_helper.py

```diff
@@ -8,11 +8,20 @@
 except ImportError:
     _hashlib = None
 
+try:
+    import _hmac
+except:
```

Comment:
?
```suggestion
except ImportError:
```

Reply:
I... think it's something I wrote fast... I'm guilty :)